android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -0,0 +1,143 @@
+package org.fossasia.openevent.adapters;
+
+import android.app.Activity;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.support.design.widget.Snackbar;
+import android.support.graphics.drawable.VectorDrawableCompat;
+import android.support.v7.widget.RecyclerView;
+import android.text.Html;
+import android.text.TextUtils;
+import android.text.method.LinkMovementMethod;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.extras.CommentItem;
+import org.fossasia.openevent.data.extras.FeedItem;
+import org.fossasia.openevent.utils.DateUtils;
+import org.fossasia.openevent.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by rohanagarwal94 on 08 /6/17.
+ */
+public class FeedAdapter extends BaseRVAdapter<FeedItem, FeedAdapter.RecyclerViewHolder> {
+
+    private Activity activity;
+    private List<FeedItem> feedItems;
+    private AdapterCallback mAdapterCallback;
+    private ArrayList<CommentItem> commentItems;
```

Comment:
List is interface, it can store ArrayList or any class implementing list, won't create any difference